CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 60.1.0
+
+* Upgrade to version 5.11.1 of govuk-frontend ([PR #4933](https://github.com/alphagov/govuk_publishing_components/pull/4933))
+
 ## 60.0.2
 
 * Style govspeak footnote targets ([PR #4992](https://github.com/alphagov/govuk_publishing_components/pull/4992))
```

Comment:
I think something's gone wrong with the CHANGELOG - 'style govspeak footnote targets' hasn't been released yet, the last release is https://github.com/alphagov/govuk_publishing_components/pull/4991